index.js

```diff
@@ -33,6 +35,23 @@ module.exports = async function standardVersion (argv) {
     }
   }
 
+  if (argv.detectRelease) {
+    const latestCommits = await getLatestCommits()
+
+    const releaseTypes = argv.types.reduce((acc, current) => (!current.hidden ? [...acc, current.type] : acc), [])
+
+    const commitSubjectTypes = latestCommits.map(c => c.subject.split(' ')[0])
+
+    const releaseAvailable = commitSubjectTypes.some(c => releaseTypes.some(type => c.includes(type)))
+
+    if (!releaseAvailable) {
+      console.info(
+        chalk.green(`No commits found for types: [ ${releaseTypes.join(', ')} ], skipping release stage.`)
+      )
+      return 0
```

Comment:
Why return 0 here? Also, I don't know if we should error out or exit successfully in this case. I'm leaned towards an error so that the process exists with != 0.

Reply:
The intention with the `return 0` was to say "script executed and exited successfully", I could replace it in favor of `process.exit()` 💭 

> Also, I don't know if we should error out or exit successfully in this case.

My perception of the issue at hand was that we didn't want to throw any error, what I understood from the initial issue was a use-case like _"I have CI that always runs `standard-version` when we a new commit is done, but it overpopulated some registry with unnecessary releases since some commits could be just `docs` or `styles` "_

I'm concerned that throwing an error could be something unnecessary alarming for the pipeline, but we can do that if other people agree.